src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs

```diff
@@ -31,6 +31,8 @@ public sealed class DownloadFromAzure : AzureConnectionStringBuildTask
 
         public string BlobNamePrefix { get; set; }
 
+        public ITaskItem[] BlobNames { get; set; }
```

Comment:
Is there a scenario were we want to get an arbitrary set of blobs that don't share a common root?